dts/bindings/vendor-prefixes.txt

```diff
@@ -55,6 +55,7 @@ ams	AMS AG
 amstaos	AMS-Taos Inc.
 analogix	Analogix Semiconductor, Inc.
 andestech	Andes Technology Corporation
+analogmicro	analog microelectronics GmbH
```

Comment:
needs to be alphabetically sorted. Also please capitalize `Analog Microelectronics` and consider fixing typo in associated commit